docs/source/reference/environment.rst

```diff
@@ -112,6 +112,13 @@ Here are the environment variables that CuPy uses at runtime.
   If set to ``1``, CuPy will use the CUDA per-thread default stream, effectively causing each host thread to automatically execute in its own stream, unless the CUDA default (``null``) stream or a user-created stream is specified.
   If set to ``0`` (default), the CUDA default (``null``) stream is used, unless the per-thread default stream (``ptds``) or a user-created stream is specified.
 
+.. envvar:: CUPY_COMPILE_WITH_PTX
+
+  Default: ``0``
+
+  If set to ``1``, CuPy will compile kernels using PTX instead of SSAS (CUBIN), PTX results in faster compilation time at the expenses of non-compatibility with drivers older than the toolkit.
```

Comment:
This option should only be used for those who *really* need this, so I want to avoid documenting it as if it is a performance tuning parameter. How about saying like this?

```
By default, CuPy directly compiles kernels into SASS (CUBIN) to support `CUDA Enhanced Compatibility <https://docs.nvidia.com/deploy/cuda-compatibility/>`_.
If set to ``1``, CuPy instead compiles kernels into PTX and let CUDA Driver assemble SASS from PTX.
This option is only effective for CUDA 11.1 or later; it will always compiles into PTX on earlier CUDA versions.
```

Question: what happens if `backend='nvcc'`?

Reply:
This is only supported for nvrtc, I will mention that in the docs